microsoftgraph/client.py

```diff
@@ -600,7 +605,7 @@ def _parse(self, response):
         if 'application/json' in response.headers['Content-Type']:
             r = response.json()
         else:
-            r = response.text
+            r = response.content
```

Comment:
Without this change the file contents returns as a `str`. When a file is downloaded as str (I tested an xlsx file), it will be corrupted. Changing to `response.content` returns `bytes` instead and the downloaded file opens correctly.